.jscsrc

```diff
@@ -1,48 +1,4 @@
 {
-  "disallowKeywords": ["with"],
-  "disallowKeywordsOnNewLine": ["else"],
-  "disallowMixedSpacesAndTabs": true,
-  "disallowMultipleVarDecl": "exceptUndefined",
-  "disallowNewlineBeforeBlockStatements": true,
-  "disallowQuotedKeysInObjects": true,
-  "disallowSpaceAfterObjectKeys": true,
-  "disallowSpaceAfterPrefixUnaryOperators": true,
-  "disallowSpacesInFunction": {
-      "beforeOpeningRoundBrace": true
-  },
-  "disallowSpacesInsideParentheses": true,
-  "disallowTrailingWhitespace": true,
-  "maximumLineLength": null,
-  "requireCamelCaseOrUpperCaseIdentifiers": true,
-  "requireCapitalizedComments": true,
-  "requireCapitalizedConstructors": true,
-  "requireCurlyBraces": true,
-  "requireSpaceAfterKeywords": [
-      "if",
-      "else",
-      "for",
-      "while",
-      "do",
-      "switch",
-      "case",
-      "return",
-      "try",
-      "catch",
-      "typeof"
-  ],
-  "requireSpaceAfterLineComment": true,
-  "requireSpaceAfterBinaryOperators": true,
-  "requireSpaceBeforeBinaryOperators": true,
-  "requireSpaceBeforeBlockStatements": true,
-  "requireSpaceBeforeObjectValues": true,
-  "requireSpacesInFunction": {
-      "beforeOpeningCurlyBrace": true
-  },
-  "requireTrailingComma": {
-      "ignoreSingleLine": true
-  },
-  "requireBlocksOnNewline": true,
-  "validateIndentation": 2,
-  "validateLineBreaks": "LF",
-  "validateQuoteMarks": "'"
+  "preset": "node-style-guide",
+  "maximumLineLength": null
```

Comment:
OK, actually, **polyjuice** isn't very interesting here since I guess it doesn't recursively parse "preset" fields.

```sh
$ polyjuice --jscs .jscsrc
{
  "rules": {
    "max-len": 0
  }
}
```

But here's an ESLint-ified version of the [JSCS "node-style-guide"](https://github.com/jscs-dev/node-jscs/blob/master/presets/node-style-guide.json) preset:
```js
{
  "rules": {
    "brace-style": 2,
    "camelcase": [2, {"properties": "never"}],
    "comma-dangle": [2, "always-multiline"],
    "curly": [2, "all"],
    "indent": [2, 2, {"SwitchCase": 1}],
    "key-spacing": [2, {"beforeColon": false, "afterColon": true}],
    "linebreak-style": [2, "unix"],
    "lines-around-comment": [2, {"afterBlockComment": true, "afterLineComment": true}],
    "max-len": [2, 80], // We override this one...
    "new-cap": 2,
    "no-mixed-spaces-and-tabs": 2,
    "no-trailing-spaces": 2,
    "no-with": 2,
    "one-var": [2, {"uninitialized": "always", "initialized": "never"}],
    "quote-props": [2, "as-needed"],
    "quotes": [2, "single"],
    "space-after-keywords": [2, "always"],
    "space-before-blocks": [2, "always"],
    "space-before-function-paren": [2, "never"],
    "space-in-parens": [2, "never"],
    "space-infix-ops": 2,
    "space-return-throw-case": 0,
    "space-unary-ops": 2
  }
}
```